homeassistant/components/sensor/filter.py

```diff
@@ -403,19 +406,23 @@ def __init__(self, window_size, precision, entity, radius):
         self._radius = radius
         self._stats_internal = Counter()
 
+    def filter_state(self, new_state, store_raw=True):
+        return super().filter_state(new_state, store_raw)
+
```

Comment:
Instead of making `store_raw` an argument of filter_state, make it a property of the filter and pass it as an argument the constructor.

Reply:
Should be fixed in 0ff0636, please review and confirm this looks ok now.